Week8/app/src/main/java/com/isabelledionisius/myomelettapp/omelettdetails/DetailActivity.kt

```diff
@@ -0,0 +1,39 @@
+package com.isabelledionisius.myomelettapp.omelettdetails
+
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.isabelledionisius.myomelettapp.omeletts.MainActivity
+import com.isabelledionisius.myomelettapp.omeletts.model.Omelett
+import com.isabelledionisius.myomelettapp.R
+import com.squareup.picasso.Picasso
+import kotlinx.android.synthetic.main.activity_detail.*
+import kotlinx.android.synthetic.main.activity_detail.view.*
+
+class DetailActivity : AppCompatActivity() {
+    /**
+     * properties to be displayed in the detail view
+     */
+    lateinit var omeletts: Omelett
```

Comment:
do you use this outside of the `assigningValues()` function?

If not, you don't need a property in the class.

Also, `assigningValues()` should be named better - maybe `displayData()` or something.